src/app/content/components/Content.tsx

```diff
@@ -26,6 +27,16 @@ import { isOpenConnector, styleWhenSidebarClosed } from './utils/sidebar';
 import Wrapper from './Wrapper';
 import { wrapperPadding } from './Wrapper';
 
+// tslint:disable-next-line:variable-name
+const GlobalStyle = createGlobalStyle`
+  html {
+    scroll-padding-top: ${(bookBannerDesktopMiniHeight + toolbarDesktopHeight) * .625}rem;
```

Comment:
` * .625` gives me anxiety, i see why it is necessary, can we extract to a constant or helper function that says what this is doing?

Reply:
I'll put it in constants.

Reply:
I guess I should ask... why is the `.625` necessary anyway?
Shouldn't rem in the `<html>` element just use its own font size?
This behavior seems... odd to me...
I'll test to see if `em` behaves better in the html element.

Reply:
Seems like using `em` instead of `rem` here removes the need for the multiplier.